Pod/Classes/PINRemoteImageManagerResult.h

```diff
@@ -7,8 +7,13 @@
 //
 
 @import UIKit;
-
+#if __has_include(<FLAnimatedImage/FLAnimatedImage.h>)
 #import <FLAnimatedImage/FLAnimatedImage.h>
+#define USE_FLANIMATED_IMAGE    1
+#else
+#define USE_FLANIMATED_IMAGE    0
+#define FLAnimatedImage         NSObject
```

Comment:
I actually copied that from here: https://github.com/pinterest/PINRemoteImage/blob/master/Pod/Classes/PINRemoteImage.h#L20

I had initially thought I was going to add an `#include PINRemoteImage.h` there to get the define, but I thought that I didn't want to pull in the extra dependencies so I added it again.